src/commands/certs/issue.ts

```diff
@@ -152,85 +152,25 @@ export default async function issue(
     cert = await createCertForCns(client, cns, contextName);
   }
 
-  if (cert instanceof ERRORS.CantSolveChallenge) {
-    output.error(
-      `We could not solve the ${cert.meta.type} challenge for domain ${
-        cert.meta.domain
-      }.`
-    );
-    if (cert.meta.type === 'dns-01') {
-      output.log(
-        `The certificate provider could not resolve the required DNS record queries.`
-      );
-      output.print(
-        '  Read more: https://err.sh/now-cli/cant-solve-challenge\n'
-      );
-    } else {
-      output.log(
-        `The certificate provider could not resolve the HTTP queries for ${
-          cert.meta.domain
-        }.`
-      );
-      output.print(
-        `  The DNS propagation may take a few minutes, please verify your settings:\n\n`
-      );
-      output.print(`  ${dnsTable([['', 'ALIAS', 'alias.zeit.co']])}\n\n`);
-      output.log(
-        `Alternatively, you can solve DNS challenges manually after running:\n`
-      );
-      output.print(
-        `  ${chalk.cyan(`now certs issue --challenge-only ${cns.join(' ')}`)}\n`
-      );
-      output.print(
-        '  Read more: https://err.sh/now-cli/cant-solve-challenge\n\n'
-      );
+  if (cert instanceof ERRORS.CertError) {
```

Comment:
What's `CertError`? Because it looks like a generic error while we are still managing fine grained errors.

Reply:
@javivelasco This is a generic cert error that indicates we can simply rely on the `error.message` to indicate what went wrong (it also has an optional `helpUrl`, making it slightly different than a plain `Error` instance)

Reply:
`wildcard_not_allowed` is only a special case for the `issue` command